admin.js

```
@@ -862,22 +879,40 @@ class AdminPanel {
+                      </div>
+                    </div>
+                    `
                     }
 
                     <div style="display: grid; grid-template-columns: repeat(2, 1fr); gap: 1rem;">
+                        ${
+                          // Show "Celková cena rezervace" only if no per-room breakdown was shown
+                          false
+                            ? `
                         <div>
                             <strong style="color: var(--gray-600); font-size: 0.9rem;">Celková cena rezervace:</strong>
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)